.github/workflows/call-fleet-integration-tests.yaml

```
@@ -105,10 +105,9 @@ jobs:
         shell: bash
         # transform download-path into msys2 equivalent
         run: |
-            DP="/tmp/fluent-bit"
+            DP="${{ steps.download.outputs.download-path }}"
```

This fails linting for some reason - I'm guessing some delta in the Windows action - and it's fixed anyway so just using the dedicated path for now.
```suggestion
            DP="/tmp/fluent-bit"
```

We should switch to the simpler action anyway.